apps/ptg-angular-app/src/app/common/common.theme.scss

```diff
@@ -11,24 +11,23 @@
 
 @mixin sidenav-theme($main, $color: #6f90b8) {
     .sidenav {
-      background-color: #F5F6F8;
+      background-color: #FFFF;
     }
     .sidenav .menu-item {
       color: $color;
       &:hover,
       &.active {
         &:not(.parent-item):not(.disabled) {
-          // background-image: url("assets/Path_bg.svg");
-          // background-size: 100% 100%;
-          background-color: $side-menu-text-color;
+
+          background-color: rgb(237, 231, 246) !important;
           border-end-end-radius: 5rem;
           border-start-end-radius: 5rem;
-          color: $white;
+          color: rgb(103, 58, 183) !important;
```

Comment:
why we are using rgb instead scss variables ?

Reply:
resolved